src/main/java/org/jabref/gui/JabRefDialogService.java

```diff
@@ -328,7 +328,7 @@ public boolean showPrintDialog(PrinterJob job) {
 
     @Override
     public Optional<Path> showFileOpenFromArchiveDialog(Path archivePath) throws IOException {
-        try (FileSystem zipFile = FileSystems.newFileSystem(archivePath, null)) {
+        try (FileSystem zipFile = FileSystems.newFileSystem(archivePath, (ClassLoader) null)) {
```

Comment:
I have Java13 running locally. With plain `null`, it does not compile, because the new method signature also allows a `Map` here. Java 12 does not. So `Map.of()` does not help.